### ISSUE TITLE
SC1097 for next 'RUN' after using 'ARG='

### ISSUE THREAD

**Issue by balrok** [hadolint/hadolint#510]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
No lint error

### Actual behavior
`Dockerfile:3 SC1097 Unexpected ==. For assignment, use =. For comparison, use [/[[.`

### Steps to reproduce the behavior
```Dockerfile
FROM ubuntu:1
ARG TEST=
RUN echo
```

Interesting:
* when using `ENV` instead of `ARG` there is no lint-error.
* when putting `COPY` before `RUN`, the SC1097 still only happens on the `RUN`-command
* when using `ARG TEST` it works -> but still this issue is quite confusing as the error does not indicate a problem with the ARG

Version:
`Haskell Dockerfile Linter v1.18.0-0-g76eee5c`

**Comment by lorenzo** [comment]
@m-ildefons can you submit a pr bumping the version for language-docker and hadolint itself?
